src/olympia/shelves/tests/test_serializers.py

```diff
@@ -43,8 +51,12 @@ def test_shelf_serializer_search(self):
         assert data == {
             'title': 'Populâr themes',
             'url': search_url,
+            'endpoint': self.search_shelf.endpoint,
+            'criteria': self.search_shelf.criteria,
             'footer_text': 'See more populâr themes',
-            'footer_pathname': ''}
+            'footer_pathname': '',
+            'addons': AddonSearchView(request=self.request).data
```

Comment:
this is fine if we're testing `AddonSearchView` elsewhere but I don't see we are.  So either have a test for `AddonSearchView` or explicitly show the output here.  (I think it's fine to `mock` `AddonSearchView` and set `.data` if there's a complete test in test_views (see other comment))